src/ctypes/ctypes_complex_compatibility.h

```diff
@@ -37,114 +38,150 @@ union ctypes_complex_float_union {
+static inline longdoublecomplex_t ctypes_compat_cpowl(longdoublecomplex_t x, longdoublecomplex_t z)
 { return cpowl(x, z); }
 
-static inline long double _Complex ctypes_compat_csqrtl(long double _Complex z)
+static inline longdoublecomplex_t ctypes_compat_csqrtl(longdoublecomplex_t z)
 { return csqrtl(z); }
-static inline long double ctypes_compat_cargl(long double _Complex z)
+static inline long double ctypes_compat_cargl(longdoublecomplex_t z)
 { return cargl(z); }
 
-static inline double ctypes_compat_creal(double _Complex z)
+static inline double ctypes_compat_creal(doublecomplex_t z)
 { return creal(z); }
-static inline double ctypes_compat_cimag(double _Complex z)
+static inline double ctypes_compat_cimag(doublecomplex_t z)
 { return cimag(z); }
-static inline double _Complex ctypes_compat_conj(double _Complex z)
+static inline doublecomplex_t ctypes_compat_conj(doublecomplex_t z)
 { return conj(z); }
 
-static inline float ctypes_compat_crealf(float _Complex z)
+static inline float ctypes_compat_crealf(floatcomplex_t z)
 { return crealf(z); }
-static inline float ctypes_compat_cimagf(float _Complex z)
+static inline float ctypes_compat_cimagf(floatcomplex_t z)
 { return cimagf(z); }
-static inline float _Complex ctypes_compat_conjf(float _Complex z)
+static inline floatcomplex_t ctypes_compat_conjf(floatcomplex_t z)
 { return conjf(z); }
 
 #if !defined(CMPLXF) || !defined(CMPLX) || !defined(CMPLXL)
 #define CTYPES_USE_STRUCT_BUILDER 1
 #else
-static inline double _Complex ctypes_compat_make_complex(double re, double im)
+static inline doublecomplex_t ctypes_compat_make_complex(double re, double im)
 { return (CMPLX(re,im)); }
-static inline long double _Complex ctypes_compat_make_complexl(long double re, long double im)
+static inline longdoublecomplex_t ctypes_compat_make_complexl(long double re, long double im)
 { return (CMPLXL(re,im)); }
-static inline float _Complex ctypes_compat_make_complexf(float re, float im)
+static inline floatcomplex_t ctypes_compat_make_complexf(float re, float im)
 { return (CMPLXF(re,im)); }
 #endif
 
 #endif
 
 #ifdef CTYPES_USE_STRUCT_BUILDER
-static inline double _Complex ctypes_compat_make_complex(double re, double im)
+static inline doublecomplex_t ctypes_compat_make_complex(double re, double im)
 { union ctypes_complex_double_union u; u.parts[0] = re; u.parts[1] = im; return u.z; }
-static inline float _Complex ctypes_compat_make_complexf(float re, float im)
+static inline floatcomplex_t ctypes_compat_make_complexf(float re, float im)
 { union ctypes_complex_float_union u; u.parts[0] = re; u.parts[1] = im; return u.z; }
-static inline long double _Complex ctypes_compat_make_complexl(long double re, long double im)
+static inline longdoublecomplex_t ctypes_compat_make_complexl(long double re, long double im)
 { union ctypes_complex_long_double_union u; u.parts[0] = re; u.parts[1] = im; return u.z; }
 #undef CTYPES_USE_STRUCT_BUILDER
 #endif
 
+// Primitive arithmetic
```

Comment:
Use `/* */` comments in header files. They might be included in foreign source code that need special compile flags that are incompatible with C99 conventions.

Reply:
Ack. Thanks. Will change.